Makefile

```diff
@@ -13,8 +13,8 @@ install:
 test:
 	go test -coverprofile=coverage.out -v ./...
 
-lint: bin/golangci-lint-1.47.2
-	./bin/golangci-lint-1.47.2 run ./...
+lint: bin/golangci-lint-1.54.2
+	./bin/golangci-lint-1.54.2 run ./...
```

Comment:
there is also an action for this https://github.com/golangci/golangci-lint-action